.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Summary
+
+
+### Before & After
+| Before |  After |
+| --- |  --- |
+|  |  |
+
+
+### Issue
+#
```

Comment:
Is this `#` intentional?

Reply:
That was a placeholder to reference an issue or PR (i.e #640).

Reply:
I'm not sure that's clear from the template... maybe replace it with `sass/sass-site#...`?

Reply:
Super late on this but made a change. Let me know what your thoughts on it.